### ISSUE TITLE
Typing `.` doesn't replace the default prompt text

### ISSUE THREAD

**Issue by fwextensions** [loreanvictor/tmplr#34]
I have a basic prompt:

```yaml
  - read: outputDir
    prompt: Where do you want to create it?
    default:
      eval: '{{ pluginName | kebab-case }}'
```

If the `pluginName` is `test`, say, then it shows that as the default text.  I'd expect when typing any character that the "selected" default text will be replaced.  But if I type `.`, it gets prepended to the default text:

![image](https://github.com/loreanvictor/tmplr/assets/61631/48230cb0-543e-4d4c-b5f9-9a67fe25347f)

If I want to enter just `.`, I have to type it and then delete the rest of the default text.

**Comment by loreanvictor** [comment]
ok it seems this issue happens with all characters that aren't alphanumeric. here's a sandbox replicating the issue:

https://stackblitz.com/edit/stackblitz-starters-1gh5yu?file=.tmplr.yml

**Comment by loreanvictor** [comment]
@fwextensions this should be fixed on `0.3.1`, please re-open if you are still encountering this issue.
